plugins/coverbrowser.koplugin/listmenu.lua

```diff
@@ -612,7 +646,7 @@ function ListMenuItem:update()
                     HorizontalSpan:new{ width = Screen:scaleBySize(10) },
                     TextBoxWidget:new{
                         text = self.text .. hint,
-                        face = Font:getFace("cfont", 18),
+                        face = Font:getFace("cfont", _fontSize(18)),
                         width = dimen.w - 2 * Screen:scaleBySize(10),
```

Comment:
With a small number of items per page (4), with long filenames, this one may overflow the dimen.h, and draw on the next item height (which is just a bit ugly), or for the last item, it may draw on the footer, which won't get refreshed (which is uglyer :)
I was going to suggest to just add here `height = dimen.h` - but then the text is top aligned and no more centered.
So, I guess we'll have to go with a `while true ... if height < dimen.h then break else fontsize = fontsize -1` like we do for the authors/titles widgets.
(I can do it later, if you have enough :)

Reply:
No worries, I'll do it - with a `repeat ... until` because I prefer it for aesthetic reasons :)